### ISSUE TITLE
Feature: better accuracy to show.

### ISSUE THREAD

**Issue by lukasz4444** [Tau-ri-Dev/JSGMod-1.12.2#226]
**Is your feature request related to a problem? Please describe.**
No it isn't.

**Describe the solution you'd like**
Ability to dial universe stargate using 7 chevron address if it is not in group mentioned in #225 

But please do not add this as forced feature but more like as an optional config option.

Because it is just an addon for #225 Idea.

**Additional context**
please watch [this video](https://youtu.be/Ri8fz7Ls8wMl) **for full context.**

**Comment by Fredyman95** [comment]
Similiar function is already in game... that universe gates can dial another dimension with only 7 symbols + PoO. You need to allow it in config...


Also videos like these are only speculative...  e.g what if...

**Comment by lukasz4444** [comment]
> Similiar function is already in game... that universe gates can dial another dimension with only 7 symbols + PoO. You need to allow it in config...
> 
> Also videos like these are only speculative... e.g what if...

Yes but idea is so it is required to input 9 symbols if gate from outside of group is dialing the universe ov group.
